umodbus/__init__.py

```diff
@@ -2,3 +2,5 @@
 # -*- coding: UTF-8 -*-
 
 from .version import __version__
+
+__all__ = ["__version__"]
```

Comment:
what is the benefit of this change?

Reply:
It's mainly to prevent flake8 errors. VS Code kept pointing out that `__version__` was unused, so keeping it in `__all__` signals that it's meant to be used as an import.